<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Trainer</class>
 <widget class="QDialog" name="Trainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trainer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="6">
    <widget class="QLabel" name="question">
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>No words</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="6">
    <widget class="QProgressBar" name="progress">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="5">
    <widget class="QLineEdit" name="answer">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="6">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QLabel" name="status">
     <property name="text">
      <string>Status</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="6">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3" colspan="2">
    <widget class="QPushButton" name="btnHint">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Hint</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="6">
    <widget class="QLabel" name="statusBar">
     <property name="text">
      <string>Progress</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QPushButton" name="btnOK">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QPushButton" name="btnAnswer">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>&amp;Answer</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="reverseMode">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Reverse mode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnAnswer</sender>
   <signal>clicked()</signal>
   <receiver>Trainer</receiver>
   <slot>giveAnswer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnHint</sender>
   <signal>clicked()</signal>
   <receiver>Trainer</receiver>
   <slot>giveHint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>Trainer</receiver>
   <slot>answerGiven()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reverseMode</sender>
   <signal>toggled(bool)</signal>
   <receiver>Trainer</receiver>
   <slot>reverseModeActivated(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>answerGiven()</slot>
  <slot>reverseModeActivated(bool)</slot>
  <slot>giveHint()</slot>
  <slot>giveAnswer()</slot>
 </slots>
</ui>
